### ISSUE TITLE
Nameerror: trainer is nit defeined

### ISSUE THREAD

**Issue by abdallah1989203** [Deci-AI/super-gradients#1157]
### 🐛 Describe the bug

![Bildschirmfoto 2023-06-11 um 15 52 50](https://github.com/Deci-AI/super-gradients/assets/49979501/30df21cb-45ff-4609-b7a4-491081888fc2)

### Versions

I have followed all the steps correctly, but it still shows me this error message!

**Comment by NatanBagrov** [comment]
Hello, could you please share the source of this example?
It looks like `trainer` was not initialized (missing `trainer = Trainer(...)`)

**Comment by abdallah1989203** [comment]
I am working on this example for a private project, so the Trainer is already present.
![Bildschirmfoto 2023-06-11 um 16 06 10](https://github.com/Deci-AI/super-gradients/assets/49979501/ad88ed70-8270-476f-a7ef-6796e75ff392)

**Comment by NatanBagrov** [comment]
To call `train(...)` you need to hold an instance of `Trainer`.
For example:
```python
...
trainer = Trainer(experiment_name=cfg.experiment_name,
                      ckpt_root_dir=get_param(cfg, 'ckpt_root_dir', default_val=None))
...
trainer.train(
        model=model,
        train_loader=train_dataloader,
        valid_loader=val_dataloader,
        training_params=training_hyperparams,
    )
...
```
